src/devtools/packages/devtools-reps/object-inspector/utils/container.tsx

```diff
@@ -22,7 +24,13 @@ export class ContainerItem implements IItem {
   }
 
   getChildren(): Item[] {
-    return this.contents;
+    return this.contents.map(item => (item.type === "value" ? item.recreate() : item));
```

Comment:
This looks kind of heavy handed, but I don't have a lot of context.

Reply:
The comment I just added on ObjectInspector's update strategy should clarify this.